framework/src/Volo.Abp.Auditing/Volo/Abp/Auditing/AuditLogActionInfo.cs

```diff
@@ -17,7 +16,7 @@ public class AuditLogActionInfo : IHasExtraProperties
 
     public int ExecutionDuration { get; set; }
 
-    public ExtraPropertyDictionary ExtraProperties { get; }
+    public ExtraPropertyDictionary ExtraProperties { get; set; }
```

Comment:
We use ExtraPropertyDictionary without setter generally. That prevents replacing existing values by mistake, developers will be only able to modify the existing collection instead of setting the entire collection with a new one

Reply:
This should be done:
entity.ExtraProperties =
{
    { "Title" , "hello-world" }
}

instead of this:
entity.ExtraProperties = new ExtraPropertyDictionary
{
    { "Title" , "hello-world" }
}